appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js

```diff
@@ -943,6 +974,48 @@ describe('appeal-details', () => {
 				);
 			});
 
+			it('should render a costs decision row in the case documentation accordion with empty status and received columns, and "add" action button, if the costs decision documents folder is empty', async () => {
+				const appealId = 2;
+				nock('http://test/').get(`/appeals/${appealId}`).reply(200, appealData);
+
+				const response = await request.get(`${baseUrl}/${appealId}`);
+
+				expect(response.statusCode).toBe(200);
+
+				const element = parseHtml(response.text);
+
+				expect(element.innerHTML).toMatchSnapshot();
```

Comment:
Don't think you need a whole page match given you are looking for the elements you need

Reply:
It's a fair point - if we're taking this approach going forward, we should create a tech debt ticket for addressing other instances of the same pattern (tests which render snapshots but also check for specific HTML), as I'm sure there are other instances elsewhere in the codebase